live-site/components/welcome-section/team-race/Team.styles.ts

```diff
@@ -0,0 +1,59 @@
+import styled from 'styled-components';
+import { H3, P } from '../../../../shared-ui/style/typography';
+import { colors } from '../../../../shared-ui/style/colors';
+import { max } from '../../../../shared-ui/lib/responsive';
+
+const StyledTeamContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  @media ${max.tablet} {
+    flex-direction: column;
+  }
+`;
+
+const StyledTeamHeader = styled(H3)`
+  color: ${colors.WHITE};
+`;
+
+const StyledTeamName = styled(P)`
+  text-transform: uppercase;
+  color: ${colors.BUTTON_GREEN};
+  font-size: 1.5em;
+`;
+
+const StyledTeamPoint = styled(P)`
+  text-transform: uppercase;
+  color: ${colors.RACE};
+  font-size: 1.5em;
+`;
+
+const StyledTeamPosition = styled(P)`
+  text-transform: uppercase;
+  color: ${colors.BUTTON_GREEN};
+  font-size: 1.5em;
+  margin-right: 0.1em;
+`;
+
+const StyledTeamCarContainer = styled.div`
+  border-bottom: 2px solid ${colors.RACE};
+  width: 15em;
+`;
+
+const StyledCarContainer = styled.div`
+  display: flex;
+`;
+
+const StyledTeamCar = styled.img<{ margin: number }>`
+  margin-left: ${(props): number => props.margin}%;
```

Comment:
Can we add like margin-bottom: -0.6em here? Right now the car is like floating but on the figma its on the line
<img width="112" alt="Screenshot 2023-01-23 at 12 11 49 AM" src="https://user-images.githubusercontent.com/59738880/213969854-14f1c937-67b3-42bd-962d-a262cc523173.png">
<img width="173" alt="Screenshot 2023-01-23 at 12 12 10 AM" src="https://user-images.githubusercontent.com/59738880/213969889-d68c22e9-d5e1-4137-a388-a927f46532eb.png">